rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMService.java

```diff
@@ -330,4 +330,12 @@ String getNodeMBeansInfo(String sessionId, String nodeJmxUrl, String objectNames
     String executeNodeScript(String sessionId, String script, String engine, String nodeUrl)
             throws RestServerException, ServiceException;
 
+    /**
+     * Retrieves the thread dump of the node identified by the given URL
+     *
+     * @param sessionId current session
+     * @param nodeUrl node to ask the thread dump from
+     */
+    String getNodeThreadDump(String sessionId, String nodeUrl) throws ServiceException, RestServerException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)